website/.circleci/config.yml

```diff
@@ -0,0 +1,10 @@
+# Circle CI 2.0 Config File
```

Comment:
this config file is not necessary, can you remove it?

Reply:
@thymikee removed.